src/components/BackToHomeHelperText/BackToHomeHelperText.tsx

```diff
@@ -0,0 +1,24 @@
+import clsx from 'clsx';
+import { Link } from 'react-router-dom';
+import { BitgoEnv } from '~/helpers';
+
+export type BackToHomeHeaderProps = {
+  env?: BitgoEnv;
+};
+
+export function BackToHomeHelperText({ env }: BackToHomeHeaderProps) {
+  return (
+    <div>
+      <span className="tw-text-gray-700">
+        Current environment: &nbsp;
+        <span className="tw-font-semibold">
+          {env === 'prod' ? 'Mainnet' : 'Testnet'}
+        </span>
+        . &nbsp;
+      </span>
+      <Link to="/" className={clsx('tw-text-blue-200', 'hover:tw-underline')}>
```

Comment:
is `clsx` needed here? looks like just `'class1 class2'` would work in this case no?

Reply:
This PR isn't changing how the component works, just making it re-useable.

Reply:
I can take a look if its easy enough to do, but if its not Ill leave that out of this PR